report/distributionList.go

```diff
@@ -0,0 +1,52 @@
+package report
+
+import (
+	"io/ioutil"
+
+	"github.com/davecgh/go-spew/spew"
+	"gopkg.in/yaml.v2"
+)
+
+// DistributionFormat is for unmarshiling a email distributionList file
+type DistributionFormat struct {
+	Severity SeverityDistribution `yaml:"severity"`
+}
+
+// SeverityDistribution list of emails ordered by severity
+type SeverityDistribution struct {
+	Debug []string `yaml:"debug,omitempty"`
+	Info  []string `yaml:"info,omitempty"`
+	Warn  []string `yaml:"warn,omitempty"`
+	Error []string `yaml:"error,omitempty"`
+	Fatal []string `yaml:"fatal,omitempty"`
+}
+
+// ReadDistributionFormatYAMLFromFile loads DistributionFormat data from a YAML file
+func ReadDistributionFormatYAMLFromFile(path string) (format DistributionFormat, err error) {
+	if data, err := ioutil.ReadFile(path); err == nil {
+		err = yaml.Unmarshal(data, &format)
+	}
+	return
```

Comment:
An excerpt from the Go Tour site:
"A return statement without arguments returns the named return values. This is known as a 'naked' return. Naked return statements should be used only in short functions, as with the example shown here. They can harm readability in longer functions."

I'll comb through the code next time we refactor; and I'll make sure all naked returns are only in short functions, 20 lines or so.